app/src/main/res/layout/profile_chooser_profile_view.xml

```diff
@@ -20,7 +20,7 @@
       type="org.oppia.android.app.model.ProfileChooserUiModel" />
   </data>
 
-  <LinearLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
We should either stay with LinearLayout in this case, or remove the inner LinearLayouts by leveraging a single outer ConstraintLayout.

In this case, I suggest the latter: we should flatten the view hierarchy here into a single ConstraintLayout.

Reply:
Seeing that this LinearLayout is also in profile_chooser_fragment.xml, and this view hierarchy has already been flattened there into a single ConstraintLayout,  I will look good if we let the LinearLayout to stay. 
please what do you suggest I do next? Thanks

Reply:
@peculiaruc I don't think you addressed this comment. To be clear: we should be optimizing this layout by flattening the hierarchy. That involves removing the nested LinearLayouts here & bringing up their elements to the new top-level ConstraintLayout, using constraints to make sure they're still presented correctly.

Note that there are also 2 other versions of this layout that need to be updated under layout-land and the sw600dp layout folders.

Reply:
Done